src/umpire/strategy/DynamicSizePool.hpp

```diff
@@ -368,16 +372,28 @@ class DynamicSizePool : private umpire::strategy::mixins::AlignedAllocation {
     return m_actual_bytes;
   }
 
+  // TODO: Keep old behaviour with m_aligned_bytes or remove and use definition
+  // from AllocationStrategy which returns m_current_size?
```

Comment:
This still needs addressing.

Reply:
I think that this overload can actually be removed, and the getCurrentSize provided by AllocationStrategy will return the current size (current total of bytes requested by user, un-aligned)

Reply:
Note that this test was updated as a result of this change: https://github.com/LLNL/Umpire/pull/908/commits/9c28b1ccef8f12414135082225c4be2758c81275.